TestProject/tests/GameLogicTest.php

```diff
@@ -0,0 +1,163 @@
+<?php
+namespace Tests;
+
+use GameOfLife\cellularAutomat\Board;
+use GameOfLife\Rules\StandardRule;
+use GameOfLife\GameLogic;
+use GameOfLife\output\ConsoleOutput;
+use Tests\CellularAutomat\BoardFromTextFiller;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Tests whether the src\GameLogic works as expected.
+ */
+class GameLogicTest extends TestCase
+{
+    /**
+     * Tests whether the calculateNextGeneration works as expected.
+     *
+     * @return void
+     */
+    public function testDoCalculateNextGenerationWithStandardRule()
+    {
+        $board = new Board(3,3);
+        $standardRule = new StandardRule();
+        $gameLogic = new GameLogic($standardRule);
+
+        $boardLayout = <<<BOARD
+        ***
+        ---
+        ---
+BOARD;
+
+        $boardFiller = new BoardFromTextFiller();
+        $boardFiller->fillBoardFromText($board, $boardLayout);
+        $consoleOutput = new ConsoleOutput();
+        $consoleOutput->outputBoard($board);
```

Comment:
Please comment this out. Unit-Tests should be silent and not produce any output. For debugging purposes it is ok to temporarily add output, but if everything works as expected you should remove it or comment it.
**Please also do this for all other occurrences in all tests!**